torch/_dynamo/eval_frame.py

```diff
@@ -884,6 +888,16 @@ def graph_with_interpreter(*args):
         graph,
     ).transform()
 
+    new_graph.meta["example_inputs"] = example_inputs
+    new_graph.meta["input_shape_constraints"] = constraints
+    new_graph.meta["inline_constraints"] = {
+        node.meta["val"].node.expr: var_to_range_map[node.meta["val"].node.expr]
+        for node in new_graph.graph.nodes
+        if node.op != "placeholder"
+        and "val" in node.meta
+        and re.match(r"^i\d+$", str(node.meta["val"]))
```

Comment:
@ezyang qq about unbacked symints: Are they always named like "i0", "i1", "i2", etc.?

Reply:
Yes but you're not supposed to rely on this...

Reply:
OK, then what is the recommended way to find constraint from unbacked symint?